Makefile

```diff
@@ -33,17 +49,33 @@ help:
 .PHONY: all
 all: html
 
+$(OUTDIR)/inbox/%: build/inbox
+build/inbox:
```

Comment:
This should be `$(OUTDIR)/inbox` or, alternatively, tweak the `$(OUTDIR):` rule to `mkdir -p $(OUTDIR)/inbox` and always create both and remove this